core/arch/arm/plat-imx/drivers/imx_caam.c

```diff
@@ -82,13 +75,13 @@ static void mkvb_init_jr(struct imx_mkvb *mkvb)
 
 	imx_caam_reset_jr(ctrl);
 	mkvb->njobs = 4;
-	io_write32((vaddr_t)&ctrl->jrstartr, 1);
-	io_write32((vaddr_t)&ctrl->jrcfg[1].irbar_ls,
+	io_write32((vaddr_t)&ctrl->jrstartr, 2);
```

Comment:
This change `1` into `2`. Please crosscheck it is what you expect.

Reply:
Yes, this was a bug previously, luckily the bootloader started the second job ring as well as the first. Thanks for noticing :-)